src/hooks/usePendingActions.ts

```diff
@@ -0,0 +1,75 @@
+import useChainId from '@/hooks/useChainId'
+import useOwnedSafes from '@/hooks/useOwnedSafes'
+import useSafeInfo from '@/hooks/useSafeInfo'
+import useTxQueue from '@/hooks/useTxQueue'
+import useWallet from '@/hooks/wallets/useWallet'
+import { Errors, logError } from '@/services/exceptions'
+import { useAppSelector } from '@/store'
+import { selectAllAddedSafes } from '@/store/addedSafesSlice'
+import { sameAddress } from '@/utils/addresses'
+import type { PendingActionsByChain } from '@/utils/queuedTxsActions'
+import { addPendingActionsByChain } from '@/utils/queuedTxsActions'
+import { getTransactionQueue } from '@safe-global/safe-gateway-typescript-sdk'
+import { useEffect, useRef, useState } from 'react'
+
+const usePendingActions = () => {
+  const addedSafes = useAppSelector(selectAllAddedSafes)
+  const ownedSafes = useOwnedSafes()
```

Comment:
This hook makes a request. Which results in two requests, one from the safe list and one from here.

I would suggest not to use it, and infer owned safes from the list of owners of each added safe. Or pass it from the safe list.
<img width="416" alt="Screenshot 2023-01-25 at 21 07 19" src="https://user-images.githubusercontent.com/381895/214678079-c27c8b92-1dd4-4eaa-8a82-949a136842d1.png">

Reply:
Good catch! I will adjust it